Changed "Game Stack Discord channel" to "Game Dev Discord channel"

v-alexmandel opened the pull request.

v-alexmandel committed:
Changed "Game Stack Discord channel" to "Game Dev Discord channel"

joannaleecy commented on azure-gaming-pr/game-dev-virtual-machine/troubleshoot-support.md:
```diff
@@ -12,7 +12,7 @@ ms.prod: azure-gaming
 The Game Development Virtual Machine is built on top of common Windows images currently available in Azure, which allows you to use much of the same troubleshooting guidance found for general Azure VMs. For more details about how to get support with your Azure VMs, please visit the [Azure support FAQ](https://azure.microsoft.com/support/faq/#support-overview).
 
-We are very eager to hear your feedback and invite you to share your experiences and feedback in our <a href="https://forms.office.com/r/VHK5iEqeBm" target="_blank">survey</a>; however, be sure to [create a ticket](https://azure.microsoft.com/support/create-ticket/) in the Azure Portal if you run into issues that require support to help unblock you. Additionally, join our <a href="https://discord ...
+We are very eager to hear your feedback and invite you to share your experiences and feedback in our <a href="https://forms.office.com/r/VHK5iEqeBm" target="_blank">survey</a>; however, be sure to [create a ticket](https://azure.microsoft.com/support/create-ticket/) in the Azure Portal if you run into issues that require support to help unblock you. Additionally, join our <a href="https://discord ...
```
Could you also update the href to : https://aka.ms/msftgamedevdiscord?

v-alexmandel committed:
Updated the href to the Discord channel

From
https://discord.com/channels/684463257276121192/949072214689452083 
to
https://aka.ms/msftgamedevdiscord

joannaleecy committed:
Merge branch 'docs' into docs
